.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -27,9 +27,9 @@ body:
       description: Which server version version you using? If your server version is not listed, it is not supported. Update to a supported version first.
       multiple: false
       options:
+        - '1.21'
         - '1.20.5/6'
         - '1.20'
```

Comment:
1.20.2 I guess?

Reply:
I changed it to mention only the versions we actually support now (unless I missed something)